build-logic/jvm/src/main/kotlin/build-logic.build-info.gradle.kts

```diff
@@ -0,0 +1,15 @@
+import buildlogic.BuildInfoTask
+import gradle.kotlin.dsl.accessors._8c47cae829ea3d03260d5ff13fb2398e.main
```

Comment:
Please remove this accidental import :)

```suggestion
```

Reply:
good catch, also had to switch from `java-base` to `java` to have access to the source sets